hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java

```diff
@@ -181,7 +181,7 @@
 
     PREFER_IPv4_STACK("hazelcast.prefer.ipv4.stack", true),
 
-    VERSION_CHECK_ENABLED("hazelcast.version.check.enabled", true),
+    PHONE_HOME_ENABLED("hazelcast.phone.home.enabled", true),
```

Comment:
I've already been making the related changes. I will share the updates with you @kobalski.